qt/python/mantidqtinterfaces/test/sample_transmission_calculator/stc_model_test.py

```diff
@@ -98,7 +98,7 @@ def test_validate_histogram_single_max_lower_than_min(self):
         input_dict = {
             "binning_type": 0,
             "single_low": 1.0,
-            "single_width": 0.1,
+            "single_width": 0.0,
```

Comment:
What was the reason for changing this value?

Reply:
The test name suggests that it's checking max < min, but it also had width > max, which would have generated two errors, so I added a separate test for checking the width > max case. The options for this case to make the test pass were to either change the width or change the expected error message, so I changed the width.